src/tools/storage-client/src/NebulaStorageClient.h

```diff
@@ -0,0 +1,121 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef NEBULA_STORAGE_CLIENT_STORAGECLIENT_H_
+#define NEBULA_STORAGE_CLIENT_STORAGECLIENT_H_
+
+#include <common/filter/Expressions.h>
```

Comment:
Be careful if you include "Expressions.h",  there are lots of other headers inside it.

Reply:
> Be careful if you include "Expressions.h", there are lots of other headers inside it.

Yes, Here is a puzzle. 
Actually I've thought encapsulating Expression into a public header file. But give it up Because it's too complicated, so just give it up.
Let me think a better way.